dts/arm/raspberrypi/rpi_pico/rp2350.dtsi

```diff
@@ -435,4 +435,12 @@
 		compatible = "raspberrypi,pico-temp";
 		status = "disabled";
 	};
+
+	id_code: id_code@10000000 {
```

Comment:
should this have image def flash in the name?

Reply:
@thedjnK I had that same debate as well when reviewing.  My understanding was the `id_code` naming was the shorthand for `ImageDef_CODE` (apologies for the caps mess) which covered the same naming area.

Reply:
I would also like to see this renamed. I wrote the image definition block. ID is a common abbreviation for with "identifier/identicification", and this isn't code (or a code, or encoding...).

It is a `Block` of metadata (sometimes referred to as a `PICOBIN_BLOCK`)  of type Image Definition, referred to as an `IMAGE_DEF` throughout the vendor's documentation.

Reply:
This was my careless mistake.
It's common to match the label here to the region name.